bin/combinetf2_fit.py

```diff
@@ -573,11 +508,19 @@ def main():
     global logger
     logger = logging.setup_logger(__file__, args.verbose, args.noColorLogger)
 
-    indata = inputdata.FitInputData(
-        args.filename, args.pseudoData, normalize=args.normalize
-    )
+    indata = inputdata.FitInputData(args.filename, args.pseudoData)
     ifitter = fitter.Fitter(indata, args)
 
+    # models for observables
+    models = [pm.models["basemodel"](indata)]
```

Comment:
See earlier comment about accomplishing this via default arguments.

Reply:
Done